src/docs/product/performance/mobile-vitals.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: "Mobile Vitals"
+sidebar_order: 60
+description: "Learn more about Mobile Vitals in Sentry, and how they give you a better understanding of your mobile application's health."
+---
+
+Mobile Vitals are a set of metrics to measure how fast your app starts, the duration of HTTP requests, the number of slow and frozen frames, and more. Each metric provides insights into the overall performance of your mobile application. These metrics give you the details you need to not only prioritize critical performance issues, but to trace issues down to the root cause to solve them faster.
+
+Mobile Vitals are summarized in several graphs on the **Performance** page in [sentry.io](https://sentry.io) when you select "Mobile" in the "Display" dropdown.
+
+![Performance page in Sentry with Mobile Vitals.](mobile-vitals.png)
```

Comment:
Where did these screenshots come from? We might want one with more data than is present in these ones.

Reply:
I took these from a Notion doc. Can you help with getting better screenshots?